template/src/ontology/Makefile.jinja2

```diff
@@ -1056,6 +1060,10 @@ $(TRANSLATIONSDIR)/%.babelon.json: $(TRANSLATIONSDIR)/%.babelon.tsv
 		convert --check false -f json -o $@.tmp.json &&\
 		mv $@.tmp.json $@
 {% endif -%}
+{% if 'db' in project.export_formats -%}
+{{ release }}.db: {{ release }}.owl
+	semsql make {{ release }}.db && rm -f {{ release }}-relation-graph.tsv.gz
```

Comment:
Here is an example of a fully battle hardened semsql pipeline with years of use:

https://github.com/monarch-initiative/mondo-ingest/blob/52541e9e0e4737168c36a17a921963675e821adc/src/ontology/mondo-ingest.Makefile#L31

```
%.db: %.owl
	@rm -f $*.db
	@rm -f .template.db
	@rm -f .template.db.tmp
	@rm -f $*-relation-graph.tsv.gz
	RUST_BACKTRACE=full semsql make $*.db -P config/prefixes.csv
	@rm -f .template.db
	@rm -f .template.db.tmp
	@rm -f $*-relation-graph.tsv.gz
	@test -f $*.db || (echo "Error: File not found!" && exit 1)
```

1. all the `rm` we added were from experience when the db was _partially created_. As the pipeline in its inner workings uses `make`, failures could have quite weird effects on caching.
2. `-P config/prefixes.csv` without this, the semsql was virtually unusable, as there were always missing prefixes. However, getting `config/prefixes.csv` right is not easy.
3. `RUST_BACKTRACE=full` I don't remember but somehow this was usfull for debugging?

Pick and choose, I am fine with what you did, just wanted to let you know what I usually do in case any of it is of any relevance. (For really due diligence you could add all the temporary files to .gitignore as well)

Reply:
> all the rm we added were from experience when the db was partially created.

😱 Makes me wonder if `semsql` is robust enough to be enabled by default… Maybe it should only be enabled by projects that can rely on “ontology pipeline engineers” being available to take care of it.

Reply:
If any case I like the idea of having a single pattern rule to handle _all_ productions of `.db` files, instead of having a similar rule at two or three different places.

Reply:
> -P config/prefixes.csv without this, the semsql was virtually unusable, as there were always missing prefixes. However, getting `config/prefixes.csv` right is not easy.

Sounds like another reason for maybe leaving the decision to enable SemSQL export to each project. Not sure how the ODK could automatically make sure the prefix file is always “right“.

Reply:
I mean, what good would it do to have all ontologies producing their own SQLite files, if said files end up being unusable and people conclude that they would be better off continuing to use the files from the “SemSQL collection”?

Reply:
I’ll leave it by default _for now_, as this has the nice side effect that SemSQL export is tested with _all_ the test ontologies from the test suite, instead of only the one in which I explicitly added the `db` export format.

But I will remove it as a default export format once the PR is finalised.